res/values-tr/strings.xml

```diff
@@ -1389,6 +1389,7 @@ Oturum açmak istemiyorsanız, tek oyunculu modumuzu oynamaya devam edebilirsini
     <string name="Account_must_be_1_day_old_to_send_mail_">Posta gönderebilmeniz için hesabınızın en az 1 günlük olması gerekmektedir.</string>
     <string name="Translator">Çevirmen</string>
     <string name="SUCCESS">BAŞARI</string>
+    <string name="PLAYER NOT FOUND">"OYUNCU BULUNAMADI"</string>
```

Comment:
It must contain the _, like: `PLAYER_NOT_FOUND`. Also, you need to either submit an [**issue**](https://github.com/simplicialsoftware/nebulous-translations/issues) if this string name doesn't exist in any of the files, or look yourself for the string at `res/values` (English file) and try adding at the correct line if it appears, so it may work well in-game, and it usually contains `translatable="false"`, as it says on **Tips**. ^^